assembl/tasks/translate.py

```diff
@@ -37,8 +57,12 @@ def __init__(self, service, discussion):
         self.base_languages.discard(None)
 
     def languages_for(self, locale_code, db=None):
+        content_locale = set()
         locale_code = self.service.asKnownLocale(locale_code)
-        return self.base_languages - set(locale_code)
```

Comment:
is asKnownLocale returning null a real case? When? Then maybe it should be debugged upstream.

Reply:
yes, it was `zxx` and `und`